client/helpers/render-elements.helpers.tsx

```diff
@@ -244,7 +244,7 @@ const renderUsageCountForm = (e: any, data: any) => {
   const formInfo: { key: string; label: string }[] = [];
   const usageFormData = getValue(e.key, data);
 
-  if (!checkIfGridHasValues(usageFormData)) return NO_DATA_LABEL;
+  if (!checkIfGridHasValues(usageFormData) || usageFormData === '-') return NO_DATA_LABEL;
```

Comment:
I think we don't even need `checkIfGridHasValues(usageFormData)` because `usageFormData === '-'` does it instead. It's much simpler, so I believe we can remove this function.

Reply:
the older forms do return the object with empty values, the newer ones don't return it at all, I don't know if some changes were made to it that removes the grid container it was in or something, but we need it for the older forms